test/errors/duplicateEnumElement.1.good

```diff
@@ -0,0 +1,23 @@
+─── error in test/errors/duplicateEnumElement.chpl:5 [MultipleEnumElems] ───
+  Enum 'languagesILike' has multiple elements named 'chapel'.
+      |
+    5 | var chpl = languagesILike.chapel;
+      |            ⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺  
+      |
+  One instance occurs here:
+      |
+    2 |   chapel, cpp, haskell,
+      |   ⎺⎺⎺⎺⎺⎺                
+      |
+  One instance occurs here:
```

Comment:
This seems OK, but if it were easy to make this into "Another instance", I think that could be even better.

Reply:
Done :)